sonar-db/src/main/java/org/sonar/db/metric/MetricDao.java

```diff
@@ -132,13 +121,12 @@ private static MetricMapper mapper(DbSession session) {
   }
 
   public void disableCustomByIds(final DbSession session, List<Integer> ids) {
-    DatabaseUtils.executeLargeInputsWithoutOutput(ids, new Function<List<Integer>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Integer> input) {
+    executeLargeInputsWithoutOutput(
+      ids,
+      input -> {
         mapper(session).disableByIds(input);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)